modules/openthread/platform/diag.c

```diff
@@ -276,3 +285,30 @@ otError otPlatDiagGpioGetMode(uint32_t aGpio, otGpioMode *aMode)
 #endif /* DT_HAS_COMPAT_STATUS_OKAY(openthread_config) && \
 	* DT_NODE_HAS_PROP(DT_COMPAT_GET_ANY_STATUS_OKAY(openthread_config), diag_gpios)
 	*/
+
+static otError startModCarrier(otInstance *aInstance, uint8_t aArgsLength, char *aArgs[])
+{
+	ARG_UNUSED(aInstance);
+	ARG_UNUSED(aArgsLength);
+
+	bool enable = true;
+	uint8_t data[OT_RADIO_FRAME_MAX_SIZE + 1];
+
+#if defined(CONFIG_NRF_802154_CARRIER_FUNCTIONS)
```

Comment:
```suggestion
#if defined(CONFIG_IEEE802154_CARRIER_FUNCTIONS)
```
This part should be platform agnostic, right?

Reply:
fixed. thanks